src/app/features/post-overview/components/post-detailview/post-detailview.component.ts

```diff
@@ -26,4 +40,34 @@ export class PostDetailviewComponent implements OnInit, OnDestroy {
   ngOnDestroy(): void {
     this.routeSub.unsubscribe();
   }
```

Comment:
Hier solltest du alle Subscriptions unsubscriben, weil die sonst neue datenströme aufmachen und ressourcen verbrauchen, sobald subscribe() iwo aufgerufen wird, aber der datenstrom mit unsubscribe() nicht zugemacht wird
So sollte das bei jedem unsubscribe aussehen (außer beim routeSub, da braucht man das '?' nicht, das '?' steht für optional, weil es diese Subscription evtl nicht geben kann, wenn die View destroyed wird: z.B. DetailView initialisiert sich und du klickst **nicht** auf edit und gehst zurück zur postoverview -> dann wurde der editSub ja nicht erstellt und ist undefined bei onDestroy, ergo kriegst du einen Fehler unsubscribe of undefined) :)
```ts
    this.dialogSub?.unsubscribe();
```